src/routes/blog/[[page]]/+page.svelte

```diff
@@ -316,4 +468,113 @@
         cursor: pointer;
         pointer-events: initial;
     }
+
+    .web-interactive-tag {
+        white-space: nowrap;
+    }
+
+    .is-selected {
+        color: hsl(var(--web-color-black));
+        background: hsl(var(--web-color-white));
+    }
+
+    .overflow-x-auto {
+        scrollbar-width: none;
+        -ms-overflow-style: none;
```

Comment:
I'm not too sure about our support levels, but don't think this is needed anymore right? Current Edge should pick

Reply:
I wasn't sure too either, added it just for safety. wouldn't want a broken view on some older edge maybe. wdyt? should I remove - `ms-*`?

Reply:
I double checked with Torsten, we can remove those!